pkg/user/user_integration_test.go

```diff
@@ -206,6 +207,59 @@ func TestUserHandler(t *testing.T) {
 				client.Do(t, http.MethodDelete, "/users/"+user1ID, nil, http.StatusUnauthorized, inttest.WithAuthToken(user2Token.AccessToken))
 			}
 		})
+
+		t.Run("ResetUserPassword", func(t *testing.T) {
+			t.Parallel()
+
+			{
+				t.Log("RequestPasswordReset")
+
+				requestResetRequestBody := strings.NewReader(`{
+					"email":    "user1@dhis2.org"
+				}`)
+
+				client.Do(t, http.MethodPost, "/users/request-reset", requestResetRequestBody, http.StatusCreated, inttest.WithHeader("Content-Type", "application/json"))
+
+				user1IDInt, _ := strconv.ParseUint(user1ID, 10, 0)
```

Comment:
require.NoError(...) ?

Reply:
Everywhere `user1IDInt` is used, it's converted to a uint. Would it make sense just doing that here?